dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/util/FeatureUtil.java

```diff
@@ -0,0 +1,59 @@
+package com.ibm.ws.sip.util;
```

Comment:
Is this necessary at this point in time? I don't think we're using the public netty-1.0 feature at all are we?

Reply:
FeatureUtil.java is deleted